src/components/Buttons/DropdownButton/DropdownButton.tsx

```diff
@@ -13,8 +14,8 @@ interface DropdownOption {
 export interface DropdownButtonProps {
   buttonText: string;
   dropdownOptions: DropdownOption[];
-  type: ButtonTypeEnum.PRIMARY | ButtonTypeEnum.SECONDARY;
-  growDirection: GrowDirection.LEFT | GrowDirection.RIGHT;
+  type?: DropdownType;
```

Comment:
any reason to use type instead of enum here?

Reply:
Auto suggestion on component import